packages/framework/src/Component/Router/FriendlyUrl/FriendlyUrlGenerator.php

```diff
@@ -60,18 +76,15 @@ public function generateFromRouteCollection(
         $entityId = $parameters['id'];
         unset($parameters['id']);
 
-        try {
-            $friendlyUrl = $this->friendlyUrlRepository->getMainFriendlyUrl(
```

Comment:
Why not add caching into friendlyUrlRepository? 

right now it's not used while exporting, and in the future, cannot be used in frontend API without hassle.

Reply:
After discussion we are continuing this way in a bit different way. Thanks to Caching component from Symfony it will be easy to use in other use cases too.